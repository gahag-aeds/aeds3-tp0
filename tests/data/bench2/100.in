100 50
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 
Sub 46 78
Sum 39 71
Sub 62 94
Min 31 63
Sub 33 65
Min 33 65
Sub 46 78
Max 1 33
Add 47 79
Max 32 64
Add 33 65
Min 29 61
Add 3 35
Max 57 89
Sub 1 33
Max 44 76
Sub 47 79
Min 5 37
Sub 47 79
Sum 23 55
Sub 32 64
Sum 50 82
Sub 51 83
Min 65 97
Add 5 37
Sum 15 47
Add 65 97
Min 65 97
Add 57 89
Min 34 66
Add 21 53
Sum 35 67
Add 22 54
Max 16 48
Add 15 47
Sum 54 86
Sub 65 97
Max 60 92
Add 4 36
Sum 45 77
Sub 24 56
Max 37 69
Sub 23 55
Sum 43 75
Add 43 75
Min 40 72
Add 19 51
Max 22 54
Add 44 76
Sum 16 48
