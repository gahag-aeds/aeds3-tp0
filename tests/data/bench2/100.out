1284
0
-1
94
96
-1
97
97
6
1503
1433
-3
1527
-2
-3
1184
97
1447
98
1221
94
1112
-4
98
1671
